Tahi-project/tahi
# Features/aperta 6169 create cover editor role

JIRA issue: [APERTA-6169](https://developer.plos.org/jira/browse/APERTA-6169)
#### What this PR does:

Adds the Cover Editor role. Initializes the role the same permissions that a Handling Editor currently has. 
## Adds a `covereditor` user to seeds with the Cover Editor role, for easier testing as this new role. 
#### Code Review Tasks:

Author tasks:  
- [x] I updated the base data.yml seeds file. [instructions](https://developer.plos.org/confluence/display/TAHI/Seeds+maintenance)

Reviewer tasks:
- ~~[ ] I performed a 5 minute walkthrough of the site looking for oddities~~ (n/a)
#### How To Test

If running locally, run the following rake tasks:
`db:reset roles-and-permissions:seed`

Log in as `covereditor` (with `password`). This user is set as the Cover Editor for the seed journal (PLOS Yeti), and will have permissions on all papers associated to that journal. 

For example: Go to paper 10 (https://tahi-staging-pr-2115.herokuapp.com/papers/10), and you can see the Cover Editor has the same access that a Handling Editor would have. 
#### After the Code Review:

Author tasks:
